### ISSUE TITLE
Custom Control Models

### ISSUE THREAD

**Issue by MagmaBro123** [eccentricdevotion/TARDIS#703]
### Describe the feature request

Title. TARDIS Consoles should have custom modeled controls.

### Describe alternatives you've considered

I've noticed in your "all-in-one" branch that you're using Display Entities, and this feature could be used for something like this, since they have right/left-click detection.

**Comment by eccentricdevotion** [comment]
Yes, this happen once the all-in-one update is complete. However display entities themselves are not clickable, it will require placing invisible barrier blocks to receive player input.

**Comment by MagmaBro123** [comment]
Ah.

**Comment by eccentricdevotion** [comment]
Closing as superseded by #836
